src/api.v2/model/MetadataTrack.js

```diff
@@ -3,21 +3,49 @@ const BasicModel = require('./BasicModel');
 const sqlClient = require('../../sql/sqlClient');
 
 const tableNames = require('./tableNames');
+const { NotFoundError } = require('../../utils/responses');
 
 const sampleFields = [
   'id',
   'experiment_id',
-  'name',
-  'sample_technology',
-  'created_at',
-  'updated_at',
+  'key',
 ];
 
 class MetadataTrack extends BasicModel {
   constructor(sql = sqlClient.get()) {
     super(sql, tableNames.METADATA_TRACK, sampleFields);
   }
 
+  async createNewMetadataTrack(experimentId, key) {
+    const sampleIds = await this.sql.select(['id'])
+      .from(tableNames.SAMPLE)
+      .where({ experiment_id: experimentId });
+
+    await this.sql.transaction(async (trx) => {
+      const response = await trx
+        .insert({
+          experiment_id: experimentId,
+          key,
+        })
+        .returning(['id'])
+        .into(this.tableName);
+
+      if (sampleIds.length === 0) {
+        return;
+      }
+
+      const [{ id: metadataTrackId }] = response;
+
+      const valuesToInsert = sampleIds.map(({ id: sampleId }) => ({
+        metadata_track_id: metadataTrackId,
+        sample_id: sampleId,
+        value: 'N.A.',
```

Comment:
you can create a constant for default metadata track sample value and use it in line 42 and in line 61

Reply:
Done